src/software/ai/hl/stp/play/kickoff_enemy_play.cpp

```diff
@@ -124,6 +145,15 @@ void KickoffEnemyPlay::getNextTactics(TacticCoroutine::push_type &yield,
             }
         }
 
+        // update robot 3 to be directly between the ball and the friendly net
+        move_tactics.at(defense_position_index)
+            ->updateControlParams(calculateBlockCone(world.field().friendlyGoalpostPos(),
+                                                     world.field().friendlyGoalpostNeg(),
+                                                     world.field().centerPoint(),
+                                                     2 * ROBOT_MAX_RADIUS_METERS),
```

Comment:
Just curious why we use `2 * ROBOT_MAX_RADIUS_METERS` here? If we use a smaller value (eg. `ROBOT_MAX_RADIUS_METERS`) we could get robot 3 to come closer to the center circle.

Reply:
Good point, I think that was what I was intending but used diameter instead of radius. 

When testing, since robot 3 is right beside the centre circle which is motion constraint, it is still fine to make a polygon (as long as polygon does not overlap into the centre circle)?

Reply:
Which polygon are you referring to? When I tested it, using `ROBOT_MAX_RADIUS_METERS` allowed the robot to get closer to the center circle which I think is what we want to achieve.